src/main/java/net/mcreator/ui/MainToolBar.java

```diff
@@ -53,12 +53,13 @@
 		add(mcreator.actionRegistry.importJSONModel);
 		add(mcreator.actionRegistry.importOBJModel);
 
-		addSeparator(new Dimension(10, 4));
+		addSeparator(new Dimension(10, 5));

```

Comment:
Why did you change this?

Reply:
I thought it was like the GridLayout inside the mod element GUIs.

Reply:
This is just spacing - how many pixels wide space to add. So no need to change this, actually rever this for even spacing inside the toolbar.